test/remoteservice/remoteservice.thrift

```diff
@@ -1,3 +1,11 @@
+include "./shared.thrift"
+
+// the unused file contains types that are not used in service endpoints. These
+// types might actually be used in embedded structs here, but since the
+// generated code could potentially contain the import of an unused package in
+// the RingpopAdapter. To prevent this the generator uses `GoUnusedProtection__`
+include "./unused.thrift"
+
 service RemoteService {
-  void RemoteCall(1: string name)
+  void RemoteCall(1: shared.UUID id)
```

Comment:
Why change it from `name` to `id`? To have less changes, you could keep it name and define a `Name` type in shared.thrift (`typedef string Name`)?